.pnpm-debug.log

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
file should be in `.gitignore`

Reply:
yes Will remove it in next commit and add it to .gitignore